From 1c5a8cad50e88000cc247cc8981c3c7eb298896c Mon Sep 17 00:00:00 2001
From: Corey Lee <lee.j.corey@gmail.com>
Date: Wed, 26 Nov 2014 21:57:27 -0500
Subject: [PATCH 1/3] Added Preferences For IDE to Ignore File

Added files to the .gitignore from the preferences that were created by
Android Studio
---
 .gitignore | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/.gitignore b/.gitignore
index d84c129..a60a685 100755
--- a/.gitignore
+++ b/.gitignore
@@ -2,3 +2,5 @@
 /bin
 *.apk
 local.properties
+.idea/
+android-xbmcremote.iml
-- 
1.9.3 (Apple Git-50)

